content/docs/faq.md

```diff
@@ -77,3 +77,13 @@ npm i --save-dev @babel/preset-env@7.14
 rm -rf node_modules package-lock.json
 npm install
 ```
+
+### 3. Store nodes only keep 30 days of messages by default, what if I need to keep messages permanently?
+
+`nwaku` and `go-waku` default value for the number of days messages are kept (`--store-days`) is 30.
+It is possible for operators to run their own store node and set a higher value.
+
+We do not believe Waku Store is appropriate to handle archived messages.
```

Comment:
What do you think about revising it to a softer version like "Waku store protocol at this point does not provide a scalable solution to serve a large size archival data (as store nodes need to persist the entire history on their own local storage space), due to this, this protocol is more suitable for serving recent historical messages. Nevertheless, research on the scalability aspect has been done and documented in [here](https://github.com/vacp2p/research/milestone/8)."

The reason I am asking is that we might be able to devise protocols by which store nodes can serve archival messages without necessarily consuming a significant storage space (e.g., y outsourcing the archival data to another p2p file storage like IPFS and having store nodes act as a proxy for retrieving archival data, and also a few other ideas).